src/js/constants.js

```diff
@@ -92,11 +92,11 @@ export const DATE_SELECTION_BUTTON_CLASSES = {
  */
 export const INSIGHTS_CARDS = [
   {
-    // For the total publications card
+    // For the total journal articles card
     numerator: "is_paper",
     denominator: null,
     denominatorText: "articles",
-    info: "<p>The total number of articles published by grantees or authors at your organization.</p>"
+    info: "<p>The total number of journal articles published by grantees or authors at your organization.</p>"
```

Comment:
Not sure this one is necessary – it is just repetitive and adding _journal_ in this context doesn’t add anything.